lms/djangoapps/grades/events.py

```diff
@@ -258,3 +261,21 @@ def fire_segment_event_on_course_grade_passed_first_time(user_id, course_locator
         )
 
     log.info("Segment event fired for passed learners. Event: [{}], Data: [{}]".format(event_name, event_properties))
+
+
+def persistent_grade_summary_changed(course_id, user_id, grade):
+     # .. event_implemented_name: PERSISTENT_GRADE_SUMMARY_CHANGED
+        PERSISTENT_GRADE_SUMMARY_CHANGED.send_event(
+            grade=GradeData(
+                user_id=user_id,
+                course=CourseData(
+                    course_key=course_id,
+                ),
+                course_edited_timestamp=grade.course_edited_timestamp,
+                course_version=grade.course_version,
+                grading_policy_hash=grade.grading_policy_hash,
+                percent_grade=grade.percent_grade,
+                letter_grade=grade.letter_grade,
+                passed_timestamp=grade.passed_timestamp,
+            )
+        )
```

Comment:
I'm not sure about this. There are two types of events here, eventtracking and open edx events. Not sure how we should handle this

Reply:
Maybe we could update the module docstring, adding something like: where all events mechanisms for grades are sent. But sending the event here it's another level of abstraction that we haven't done before :thinking:

Reply:
I had not noticed, but I think I agree. We are mixing openedx-events from oep-50 with the tracking data events.

Perhaps we would be better suited calling `PERSISTENT_GRADE_SUMMARY_CHANGED.send_event` inside the `_emit_openedx_persistent_grade_summary_changed_event` function.